lib/pages/myaccount_screen.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:flutter/material.dart';
+import 'package:intern_task/pages/menu_button.dart';
+
+class MyAccountScreen extends StatefulWidget {
+  const MyAccountScreen({super.key});
+
+  @override
+  State<MyAccountScreen> createState() => _MyAccountScreenState();
+}
+
+class _MyAccountScreenState extends State<MyAccountScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Scaffold(
+        appBar: AppBar(
+            title: const Text('My Account'),
+            leading: IconButton(
+                onPressed: () {
+                  Navigator.pop(context);
+                },
+                icon: const Icon(Icons.arrow_back_ios_sharp))),
+        body: Column(
+          children: [
+            const SizedBox(height: 30),
+            Expanded(
+              flex: 1,
+              child: Container(
+                padding: const EdgeInsets.all(30),
+                decoration:
+                    BoxDecoration(color: Theme.of(context).primaryColor),
+                child: Row(
+                  children: [
+                    CircleAvatar(
+                      radius: 40,
+                      child: Image.asset(
+                        'card1.png',
+                        fit: BoxFit.fill,
+                      ),
+                    ),
+                    const SizedBox(width: 15),
+                    Column(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: const [
+                        Text(
+                          'John Doe',
+                          style: TextStyle(
+                              fontSize: 25,
+                              fontWeight: FontWeight.bold,
+                              color: Colors.white),
+                        ),
+                        Text(
+                          'John@email.com',
+                          style: TextStyle(
+                              fontSize: 12,
+                              fontWeight: FontWeight.bold,
+                              color: Colors.white),
+                        )
+                      ],
+                    )
+                  ],
+                ),
+              ),
+            ),
+            Expanded(
+              flex: 4,
+              child: Container(
+                padding: const EdgeInsets.all(30),
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    const MenuButton(buttonName: 'Account Setting'),
```

Comment:
u can also make a list and then map to a widget, contact me for clarification

Reply:
ok done